src/platform/ESP32/NetworkCommissioningDriver.h

```diff
@@ -25,7 +25,8 @@ namespace NetworkCommissioning {
 namespace {
 constexpr uint8_t kMaxWiFiNetworks                  = 1;
 constexpr uint8_t kWiFiScanNetworksTimeOutSeconds   = 10;
-constexpr uint8_t kWiFiConnectNetworkTimeoutSeconds = 30;
+constexpr uint8_t kClientCommunicationDalay         = 10;
```

Comment:
Is this bit still needed given the changes in https://github.com/project-chip/connectedhomeip/pull/19137 ?

Reply:
No its not needed now. Removed